packages/sdk/src/types/runtime/appearance.ts

```diff
@@ -19,13 +20,22 @@ export class Appearance implements AppearanceLike {
     private _materialId = ZeroGuid;
     // tslint:enable:variable-name
 
-    /** Whether the actor with this appearance should be visible */
+    /**
+     * This actor's visibility preference, independent of its parent. See
+     * [[activeAndEnabled]] for the actual visibility state.
+     */
     public enabled = true;
```

Comment:
Seems like `enabled` should be a getter now, so it can return enabledFor's packed value, if it is set.